src/fides/api/ops/service/saas_request/saas_request_override_factory.py

```diff
@@ -133,7 +133,7 @@ def validate_read_override_function(f: Callable) -> None:
     and that it declares at least 5 parameters.
     """
     sig: Signature = signature(f)
-    if sig.return_annotation is not List[Row]:
+    if sig.return_annotation != List[Row]:
```

Comment:
This was causing some failing tests on this branch.  I think a '!=' comparison is recommended here anyway. We just want to see if the values are equal, not if they are stored in the same place in memory.